lib/meta.nix

```diff
@@ -41,16 +41,18 @@ rec {
     let x = builtins.parseDrvName name; in "${x.name}-${suffix}-${x.version}");
 
 
-  /* Apply a function to each derivation and only to derivations in an attrset
+  /* Apply a function to each derivation and only to derivations in an attrset.
   */
   mapDerivationAttrset = f: set: lib.mapAttrs (name: pkg: if lib.isDerivation pkg then (f pkg) else pkg) set;
 
+  /* Set the nix-env priority of the package.
+  */
+  setPrio = priority: drv: addMetaAttrs { inherit priority; } drv;
```

Comment:
is this not basically the same as?

```
addMetaAttrs { priority = 10; } drv;
```

Reply:
I think this is a no-op readability change; do you think exposing implementation is better?

(for example, if there are other frequent `meta` overrides, improving visibility of `addMetaAttrs` could be indeed useful)